dashboard/modules/actor/tests/test_actor.py

```diff
@@ -240,9 +240,8 @@ def handle_pub_messages(client, msgs, timeout, expect_num):
 
     msgs = []
     handle_pub_messages(p, msgs, timeout, 2)
-
     # Assert we received published actor messages with state
-    # DEPENDENCIES_UNREADY and ALIVE.
+    # DEPENDENCIES_UNREADY and PENDING_CREATION.
     assert len(msgs) == 2
```

Comment:
```suggestion
    # DEPENDENCIES_UNREADY, PENDING_CREATION and ALIVE.
    assert len(msgs) == 3
```

Reply:
and change the latter one from `handle_pub_messages(p, msgs, timeout, 3)` to `handle_pub_messages(p, msgs, timeout, 4)`.